### PR TITLE
adding in hbo max config and styling additions

### PR DESCRIPTION
**Pull request by unijuglr**
**Links**

    https://wikia-inc.atlassian.net/browse/CAKE-5678

**Description**

Adding config and a slight template addition to support HBO MAX campaign
Can go live without concern, because it will be controlled by db rows being added at launch time

**Reviewers**
@Wikia/cake

### CODE REVIEW

File: extensions/wikia/AffiliateService/js/units.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+		{
+			campaign: 'hbomax',
+			category: 'hbomax',
+			name: 'hbomax-unit',
+			country: ['US'],
+			heading: 'Where HBO Meets So Much More',
+			subheading: 'START YOUR FREE TRAIL',
+			link: 'https://www.hbomax.com/gzm',
+			image: 'https://static.wikia.nocookie.net/05a5bbb9-7662-416e-99c7-6e6700af9aa3',
+			extra_disclaimer: "Free trail for new customers only. Restrictions Apply."
```
Comment (cschucode) [inline extensions/wikia/AffiliateService/js/units.js]: Looks like a typo: `trail => trial`

File: extensions/wikia/AffiliateService/styles/affiliate-unit.scss [hunk 2]
```diff
@@ -85,6 +85,16 @@
         position: relative;
     }
 
+    .WikiaArticle p &__extra-disclaimer-message {
```
Comment (cschucode) [inline extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: Looks like the same styles from `&__disclaimer-message`.  Do we need to keep these styles separate?

File: extensions/wikia/AffiliateService/js/templates.js [hunk 3]
```diff
@@ -4,18 +4,25 @@ define( 'ext.wikia.AffiliateService.templates', [], function() {
...
         }
-        
+
         if (options.showDisclaimer) {
             disclaimer = '<p class="aff-unit__disclaimer-message">'
                 + options.disclaimerMessage
                 + '</p>';
         }
 
+        if (options.extra_disclaimer) {
```
Comment (cschucode) [inline extensions/wikia/AffiliateService/js/templates.js]: Let's use stay consistent with Camel Casing. (see above)

File: extensions/wikia/AffiliateService/js/units.js [hunk 4]
```diff
@@ -1,5 +1,16 @@
 define('ext.wikia.AffiliateService.units', [], function () {
 	var units = [
+		{
+			campaign: 'hbomax',
+			category: 'hbomax',
+			name: 'hbomax-unit',
+			country: ['US'],
+			heading: 'Where HBO Meets So Much More',
+			subheading: 'START YOUR FREE TRAIL',
```
Comment (cschucode) [inline extensions/wikia/AffiliateService/js/units.js]: Another TRAIL??

**Review by cschucode — CHANGES_REQUESTED** [review]
Minor: 
* Code style
* typos

File: extensions/wikia/AffiliateService/js/units.js (hunk 1 above)
Comment (unijuglr) [reply extensions/wikia/AffiliateService/js/units.js]: No, it's Trail... as in something that you walk down to get to another place...

;-)

File: extensions/wikia/AffiliateService/styles/affiliate-unit.scss (hunk 2 above)
Comment (unijuglr) [reply extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: Potentially, I figured the flexibility to do so outweighed the assumption that we wouldn't want to...

Comment (unijuglr) [reply extensions/wikia/AffiliateService/styles/affiliate-unit.scss]: and then I just noticed that I didn't even end up using in to the template... D'oh!. Removing...

File: extensions/wikia/AffiliateService/js/units.js (hunk 4 above)
Comment (unijuglr) [reply extensions/wikia/AffiliateService/js/units.js]: Hey, at least my typos are consistent...

**Review by cschucode — APPROVED** [review]
LGTM. My only question is if we're not using the extra disclaimer, do we still need it in there?

Also, looks like a build error needs to be fixed before merging.

**Review by JoshuaRogan — APPROVED** [review]

**Pull request merged**